Formula/stress-ng.rb

```diff
@@ -0,0 +1,15 @@
+class StressNg < Formula
+  desc "Stress test a computer system in various selectable ways"
+  homepage "http://kernel.ubuntu.com/~cking/stress-ng/"
+  url "https://github.com/ColinIanKing/stress-ng/archive/V0.09.04.tar.gz"
```

Comment:
Let's use the official release tarball: http://kernel.ubuntu.com/~cking/tarballs/stress-ng/stress-ng-0.09.06.tar.xz

Reply:
Fixed in 1f78751